packages/blade/src/components/Radio/_decisions/decisions.md

```diff
@@ -0,0 +1,113 @@
+# Radio Decisions <!-- omit in toc -->
+
+Radio is used to present each item in a list of options where users must make a single selection.
+
+This document outlines the API of `Radio` & `RadioGroup` component.
+
+<img src="./radio_thumbnail.png" alt="Radio component thumbnail" width="380" />
+
+- [Design](#design)
+- [`Radio` API](#radio-api)
+- [`RadioGroup` API](#radiogroup-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+    - [Controlled, Uncontrolled](#controlled-uncontrolled)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+- [Figma Link - Radio](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13133%3A160709)
+
+## `Radio`
```

Comment:
This is also a question for design but will we ever have a single radio button? Radio buttons always exist in groups and only one of them should be selectable in which case shouldn't they be all part of a `RadioGroup`?

We had an unusual case for Checkbox where a single checkbox could exist (eg. the terms and conditions check), but I can't think of where we will have a use case for a single radio button?

Reply:
Correct we won't have single Radio outside of RadioGroup. Since a radio will only work if it's grouped together with multiple radios by binding the `name` html prop.